boards/nrf52832-mdk/include/gpio_params.h

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
just some idea for future improvement: how about we generalize `boards/common/nrf52xxxdk/include/gpio_params.h` a bit more, so we can re-use it in this and other boards? We simply would have to make the `name` (and possibly the `flags`) field 'definable' and there would be no need to duplicate this file anymore.

Reply:
I like the idea. I'll see if I can find some time to look at it if you agree.